diffs/diff4.diff

```diff
@@ -0,0 +1,108 @@
+*TEST FILE FOR DIFF*
```

Comment:
What's special with this diff file to justify its inclusion? Does it add an edge case or something?

Reply:
In the absence of unit testing I instead opted to have a smaller diff file where I could more easily visualize the number of added lines, deleted lines, and functionCalls. 
It was also helpful when it came to testing certain edge cases as well.
If this was code to be deployed, it would need a suite of unit tests to ensure functionality over time.